README.md

```diff
@@ -230,11 +231,14 @@ run HarbourBridge on the modified pg_dump/mysqldump output.
 
 HarbourBridge generates several files as it runs:
 
-- Schema file (ending in `schema.txt`): contains the generated Spanner
-  schema, interspersed with comments that cross-reference to the relevant
+- Schema DDL file (ending in `schema.ddl.txt`): contains the generated Spanner
+  schema DDL, interspersed with comments that cross-reference to the relevant
   PostgreSQL/MySQL schema definitions.
 
-- Report file (ending in `report.txt`) contains a detailed analysis of the
+- Schema conversion file (ending in `schema.conv.json`): contains all schema 
```

Comment:
The `schema.conv.json` file has a lot more than just schema. It has everything on the conv datastructure, including schema, mappings, stats, errors, statements etc -- including everything needed to generate report.txt.

It is basically the snapshot of the state of the tool -- kind of a record of the session.

How about session.json?

Reply:
Yup, sounds good. I will make that to `session.json`.